api/src/main/resources/workbench.yaml

```diff
@@ -1839,9 +1860,14 @@ definitions:
         $ref: '#/definitions/Profile'
       password:
         type: string
+
+  InvitationVerRequest:
```

Comment:
I got stuck on this, so I think we need to just spell it out as `InvitationCodeVerificationRequest`.